pkg/controller/health_check.go

```diff
@@ -6,13 +6,19 @@ import (
 	"net/http"
 
 	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/codeready-toolchain/registration-service/pkg/errors"
 	"github.com/gin-gonic/gin"
 )
 
 // HealthCheck implements the health endpoint.
 type HealthCheck struct {
-	config *configuration.Registry
-	logger *log.Logger
+	config      *configuration.Registry
+	logger      *log.Logger
+	Alive       bool
```

Comment:
We need to separate the HealthCheck controller from the Health Payload. These are two different objects.

Reply:
Yeah agreed, fixed here: be101406e7a9d5eba80a3ba6bd2e55510c51d8e8